index.html

```diff
@@ -17,13 +17,14 @@
       <header class="header">
           <h1><a href="#">My Logo</a></h1>
           <nav> 
-            <img src="images/Union.png" alt="hamburgeur menu" class="mobile-img">
-            <!-- <img src="images/desktop-header.svg" alt="hamburgeur menu" class="desktop-img"> -->
+            <a href="#"><img src="images/Union.png" alt="hamburgeur menu" class="mobile-img"></a>
+         
           </nav>
-          <ul class="header-menu">
-            <li>Portoflio</li>
-            <li>About</li>
-            <li>Contact</li>
+          <ul class="header-menu" id="header-menu">
+            <li><a href="#portfolio"  class="link">Portoflio</a></li>
+            <li><a href="#about" class="link">About</a></li>
+            <li><a href="#contact" class="link">Contact</a></li>
+            <li> <a href="#"><img src="images/removenav.svg" alt="Back hamburgeur menu " class="remove mobile-img"></a></li>
           </ul>
```

Comment:
- [ ] Good job with your mobile menu, it works as expected. So, I noticed that there is still one thing you should work on to make your milestone complete. Make The  menu follows the design from the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., **font-size**, **font-weight**, **line-height**, **spacing(margins or padding) between and around all the elements of the menu**, the **blur effect** on the background,...). Make sure you add everything to stick to the Figma template.

| Figma design | Your design |
| ------------------ | ----------------- |
| ![mmf1](https://user-images.githubusercontent.com/83042742/174842212-22a5bb0f-a33f-46bf-999e-e1acf97a4187.png) | ![mms1](https://user-images.githubusercontent.com/83042742/174843351-066b5c7d-eba8-4fed-8c29-2a366edd5645.png) |